agent/iskakalunan.py

```diff
@@ -66,7 +66,7 @@ def run(self):
             subprocess.call(['python', 'runtest.py', '-h', '>', 'result'])
             return
 
-        subprocess.call(['git', 'checkout', 'master'])
+        subprocess.call(['git', 'checkout', 'dev'])
```

Comment:
by default, this should be master as always. The purpose to move to master was for local testing. Per Shako's comment, we were expecting to push local builds/try builds to e-mail account and be tested locally. If that's the case, it should run on master.

However, we are slow in updating master. That's the reason I propose to move current working branch to dev. Supposedly, we should move back to master in the future.